### DISCUSSION TITLE
Standup (Monday, October 31st)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [BBGTC/food-bank-app#32] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Checked styles and navigation

**What are your priorities for the next 24 hours?**:
 - Define style management

**Blockers**:
 - Not sure how to manage styles between files

**Shoutouts**:
 - @antonieto for his offer at @Google (official now)

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Started test project with Django as Ive never used it before
 - Designed use case for #8 

**What are your priorities for the next 24 hours?**:
 - Finish test project
 - Do some tests with Django
 - Start implementation of #8

**Blockers**:
 - Not a lot of time

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Created isAuthenticated Hook and Context (just a refactor, since the app functionallity remains the same)

**What are your priorities for the next 24 hours?**:
 -  Check backend structure with @antonieto and start working on #28 

**Blockers**:
 - None

**Shoutouts**:
 - to every BB member for being amazing teammates

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Implemented #4
 - Implemented a django channels web socket proof of concept

**What are your priorities for the next 24 hours?**:
 - Define tasks for sprint 3

**Blockers**:
 - i'm a bit lost

**Shoutouts**:
 - @CarlosCRG19 @BrandonMagana @DavBot02 for pair programming with me last friday
